app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,46 @@
+class BookingsController < ApplicationController
+  def index
+    @bookings = policy_scope(Booking).order(created_at: :desc)
+  end
+
+  def new
+    @pet = Pet.find(params[:pet_id])
```

Comment:
I think it would be better to create a find method so that you keep the code dry and not repeat it in each method